src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala

```diff
@@ -74,6 +74,17 @@ final class ScalaSensor(globalConfig: GlobalConfig) extends Sensor {
       }
   }
 
+  def ExampleIssues(): Unit = {
```

Comment:
![info](https://static.sonar-scala.com/img/severity-info.svg 'Severity: info') Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))